tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -533,14 +540,14 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
 
     this.isPinned$ = this.store.select(getCardPinnedState, this.cardId);
 
-    this.stepSelectorTimeSelection$ = combineLatest([
+    combineLatest([
       this.minMaxSteps$,
       this.store.select(getMetricsStepSelectorEnabled),
-    ]).pipe(
-      map(([{minStep}, enableStepSelector]) => {
-        return enableStepSelector ? {start: {step: minStep}, end: null} : null;
-      })
-    );
+    ]).subscribe(([{minStep}, enableStepSelector]) => {
+      this.stepSelectorTimeSelection$.next(
```

Comment:
So, to be clear, this will reset the stepSelectorTimeSelection to the min step if minMaxSteps$ changes? I understand this already happens. I just want to make sure I understand. Also, now that this is a BehaviorSubject we can view the actual value right? So we could for instance check to see if the current value is less than the new min value or something along those lines?

Reply:
Yes this will reset to `minStep` when `minMaxSteps$` changes.
Yes we can view the value with `getValue`